commons-shared/src/main/scala/com/avsystem/commons/SharedExtensions.scala

```diff
@@ -128,4 +129,43 @@ object SharedExtensions extends SharedExtensions {
     def toNOpt: NOpt[A] =
       if (tr.isFailure) NOpt.Empty else NOpt.some(tr.get)
   }
+
+  object TryOps {
```

Comment:
I don't think `SharedExtensions` is the right place for it. `SharedExtensions` is intended to keep extension methods that can be inherited by package object (e.g. `package object ump`) so that these extension methods are visible "globally".